specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-04-01/bms.json

```diff
@@ -18,6 +18,149 @@
     "application/json"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupstorageconfig/vaultstorageconfig": {
```

Comment:
The api-version 2018-12-20 has the same path, but the tag is different. So it's conflict. Pls fix it

Reply:
Hi @ruowan In this PR the API BackupResourceStorageConfigs which is already present in the version 2018-12-20, is being added in the version 2021-04-01(which routes the request to the active stamp)  with  different operationId and tag which will ensure a new SDK function is generated for this API that routes the request specifically to active stamp.

We have followed similar procedure for a number of other APIs and the SDK generation has always worked in the past. This is an example of such a scenario : This is the API in 2021-04-01  : https://github.com/Azure/azure-rest-api-specs/blob/8dff86df71bee429af84ea4713288ef3cdd1db2f/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-04-01/bms.json#L374
This is the same API in 2018-12-20 with different operationId and tag  : https://github.com/Azure/azure-rest-api-specs/blob/8dff86df71bee429af84ea4713288ef3cdd1db2f/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json#L21

Reply:
It seems you changed the tags of the below api in the new api version , this caused conflict.
`$.paths["/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.RecoveryServices/vaults/vaultName/backupstorageconfig/vaultstorageconfig"].get`

Reply:
discussed offline, accept this change as this situation exists for historical reason, and cannot easily fix.